pkg/validator/validators.go

```diff
@@ -304,20 +305,30 @@ func (kc *validator) validateMRCOnCreate(mrc *configv1alpha2.MeshRootCertificate
 		return err
 	}
 
-	if mrc.Spec.Intent == v1alpha2.ActiveIntent {
-		foundActive, err := kc.checkForExistingActiveMRC(mrc)
+	switch mrc.Spec.Intent {
+	case configv1alpha2.ActiveIntent, configv1alpha2.PassiveIntent:
+		certsInUse, err := kc.getCertsInUseMRCs()
 		if err != nil {
 			return err
 		}
-		if foundActive {
-			return fmt.Errorf("cannot create MRC %s with intent active. An MRC with active intent already exists in the control plane namespace", getNamespacedMRC(mrc))
+		// if there is already active or passive mrc then we can't create a new one
+		if mrcWithIntentExist(certsInUse, mrc) {
```

Comment:
nit: this function could be renamed for clarity. Maybe `mrcWithMatchingIntentExists`?